backend/app/core/rag/kb_manager.py

```
@@ -1,182 +1,170 @@
+
+        if "knowledge_graph" not in kb:
+            warnings.append("Граф знаний не был построен")
+
+        return {
+            **state,
+            "chunks_count": len(kb.get("chunks", [])),
+            "warnings": warnings,
+            "errors": errors,
+        }
+
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)